src/operator/operator_tune-inl.h

```diff
@@ -56,7 +56,7 @@ namespace op {
 #endif
 #endif  // MXNET_NO_INLINE
 
-#define OUTSIDE_COUNT_SHIFT    9
```

Comment:
does changing this impact the IsOMPFaster selection faster in operator_tune.h. Do we need to tweak WORKLOAD_COUNT_SHIFT too ?

Reply:
Workload_count_shift is currently 11, which means workload count will be 2048. 
this means that operation is done for 2048 times. This number can be made 10, but IsOMPFaster doesn't look like bottleneck for the related issue. It is the function which is calculating OMP overhead which is causing the problem.